packages/core/src/applicationcomposer/messageHandlers/emitTelemetryMessageHandler.ts

```diff
@@ -26,43 +26,43 @@ export function emitTelemetryMessageHandler(message: EmitTelemetryMessage) {
     try {
         const parsedData = message.metadata ? JSON.parse(message.metadata) : {}
         switch (message.eventType) {
-            case 'GENERATE_CLICKED':
+            case TelemetryType.GENERATE_CLICKED:
```

Comment:
This is another small disadvantage of enums. If a `type` was used, the literal value could be used instead.

Reply:
Would it be better now to switch it from `enum` in to a `type` instead? It looks safe to do